ioctl/newcmd/contract/parse.go

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package contract
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-core/pkg/util/addrutil"
+	"github.com/pkg/errors"
+)
+
+// ErrInvalidArg indicates argument is invalid
+var (
+	ErrInvalidArg = fmt.Errorf("invalid argument")
+)
+
+func parseAbi(abiBytes []byte) (*abi.ABI, error) {
+	parsedAbi, err := abi.JSON(strings.NewReader(string(abiBytes)))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal abi")
+	}
+	return &parsedAbi, nil
+}
+
+func parseInput(rowInput string) (map[string]interface{}, error) {
+	var input map[string]interface{}
```

Comment:
can use struct with json tag?

Reply:
> can use struct with json tag?

i think we can't use any specific struct here, because the input may have unknown field